### ISSUE TITLE
slugify categories?

### ISSUE THREAD

**Issue by sherwinmartin** [ekynoxe/JekyllCategoriesTag#1]
thanks for the great plugin. i was wondering if there's a way to slugify the categories. i have categories that are more than 1 word, although it works as is, i'd like it to be more seo friendly as well as follow web standards. thanks again.

**Comment by ekynoxe** [comment]
Good catch. This is actually probably a bug, as, as you say, it should strip special characters and replace spaces by dashes, or something like this. I'll look into it.

**Comment by ekynoxe** [comment]
@w1n78 I've updated my other plugin for this, the category pages generator at https://github.com/ekynoxe/JekyllCategoryPagesGenerator but I realise now that this plugin creating  a tag to generate a list of categories will need more work to make it really useful.
I didn't realise until now that Jekyll generates category pages regardless, which aren't usable at all while slugifying the category links.

I think I'll have to see if I can merge those two plugins, or find another way to interact with the way Jekyll generate the category directories.

**Comment by ekynoxe** [comment]
@w1n78 Am i right thinking you commented on the wrong project? :) Our initial exchange on Twitter talked about the CategoryPagesGenerator.

**Comment by sherwinmartin** [comment]
oh wow just realized that. sorry and thanks for the update.
